src/components/Login.tsx

```diff
@@ -0,0 +1,114 @@
+import { css } from '@emotion/react';
+import theme from '@src/styles/tokens/index';
+import kakao_logo from '@src/assets/icons/kakao_logo.svg';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+const mainStyle = css`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  height: calc(-2.75rem + 100vh);
+  align-items: center;
+  justify-content: center;
+`;
+
+const logoStyle = css`
+  background-color: ${theme.colors.gray100};
```

Comment:
로고 배경 색상을 이 색상으로 지정하신 이유가 있을까요..? 😅
```suggestion
  background-color: ${theme.colors.gray00};
```